docs/advanced/1-multilevel.md

```diff
@@ -175,6 +175,122 @@ IS-IS Instance: Gandalf VRF: default
 
 You will immediately notice that R1 lost the full topological view of the network. The IS-IS database contains only LSPs originated by C1, C2, and R1.
 
+We made one step forward. Now let's have a detailed view of the LSPs originated by our L1/L2 routers. We will only examine C1 here:
```

Comment:
I would start the changed part with "now let's look at the R1 routing table. What are all those external subnets doing in it". Always start with the easy part (routing table), then start digging (into LSPs)